src/azure-cli/azure/cli/command_modules/cloud/custom.py

```diff
@@ -148,3 +148,8 @@ def list_profiles(cmd, cloud_name=None, show_all=False):
     if not cloud_name:
         cloud_name = get_active_cloud_name(cmd.cli_ctx.cloud)
     return list(API_PROFILES)
+
+
+def import_cloud_endpoints(cmd, endpoint=None):
```

Comment:
where is the parameter endpoint used?

Reply:
It's set in `__main__.py` and used in `cloud.py`, then the command execution reaches here, but the endpoints are actually already imported in AzCli init process.

Reply:
I don't see where this parameter used in this method